src/main/java/frc/robot/subsystems/drivetrain/commands/DriveAndAdjustWithVision.java

```diff
@@ -60,8 +60,8 @@ public void execute() {
         if (magnitude == 0) current = 0;
         current += magnitude / 5;
         if (current > magnitude) current = magnitude;
-        forward = Math.cos(alpha) * current;
-        strafe = Math.sin(alpha) * current;
+        forward = Math.cos(alpha) * magnitude;
+        strafe = Math.sin(alpha) * magnitude;
```

Comment:
So you don't use the `current` variable?

Reply:
Usually, it's fine. But, because we don't fully stop until wheels are adjusted, and because the speed multiplier is so high, and because of the x0.5 speed on trigger. I noticed it could cause some problems.